secure-api-gateway-helpers/readme.md

```diff
@@ -0,0 +1,53 @@
+# Secure API Gateway - Releases - Secure API Gateway Helpers
+
+## Prerequisites
```

Comment:
Overview section again please!

Reply:
Will do this one once the 'Main' Readme has been okay'd